requirements.yml

```diff
@@ -1,2 +1,5 @@
 collections:
   - pulp.pulp_installer
+  - name: https://github.com/ehelms/foreman-operations-collection
+    type: git
+    version: postgres-upgrade
```

Comment:
Needs update before merging, after the role has been merged to the collection (and released).  See https://github.com/theforeman/foreman-operations-collection/pull/21

Reply:
Also I will work on a release of the collection.